src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for reading `.deps` files.",
-  "version": "1.1.1",
+  "version": "1.1.2-servicing-*",
```

Comment:
Do we actually plan to ship this or PlatformAbstractions from the servicing branches? I just checked and we did ship it with the 1.1.1 release but I would have not expected us to given we haven't changed anything here. We should try to avoid shipping these if it isn't necessary.